### PR TITLE
    Add an initial Web developer focused landing page

### PR DESCRIPTION
**Pull request by ericholscher**
This tries to get us on Google for a few more important keywords.

### Preview

localhost_8000_publishers_topics_web-developers_

### CODE REVIEW

File: content/pages/landing-pages/web-developers.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+title: Monetize your web app or tool with privacy-first ads
```
Comment (davidfischer) [inline content/pages/landing-pages/web-developers.md:1]:
```suggestion
title: Monetize your Web App or Tool with Privacy-first Ads
```

File: ethicalads-theme/templates/ea/landing-pages/web-developers.html [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+{% block content %}
+<section class="position-relative mt-5 mb-10">
+
+  <!-- Content -->
+  <div class="container">
+    <div class="row align-items-center text-center text-md-left">
+      <div class="col-12 col-md-6">
+
+        <!-- Image -->
+        <img src="/theme/img/python-code.jpg" alt="Web App Code" class="img-fluid mw-100 float-right mb-6 mb-md-0" data-aos="fade-right">
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/landing-pages/web-developers.html:21]: I like this image and it's almost more a fit here than in Python but we probably shouldn't re-use it.

File: ethicalads-theme/templates/ea/landing-pages/web-developers.html [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+      <div class="col-12 col-md-6">
+
+        <!-- Heading -->
+        <h1 class="display-3 font-weight-light" data-aos="fade-left">Focus on features for your web app</h1>
+
+        <!-- Text -->
+        <p class="font-size-lg" data-aos="fade-left" data-aos-delay="200">Get paid to host well-designed and privacy-first ads on your site.</p>
+
+        <a href="#inbound-form" class="btn btn-primary lift" data-aos="fade-left" data-aos-delay="200">
+          <span>Apply to be a publisher today</span>
```
Comment (davidfischer) [inline ethicalads-theme/templates/ea/landing-pages/web-developers.html:38]: I don't mind this but I also think "Join our ad network" or "Apply to join our network" are good too.

File: content/pages/landing-pages/web-developers.md (hunk 1 above)
Comment (ericholscher) [reply content/pages/landing-pages/web-developers.md:1]: 🤦

File: ethicalads-theme/templates/ea/landing-pages/web-developers.html (hunk 3 above)
Comment (ericholscher) [reply ethicalads-theme/templates/ea/landing-pages/web-developers.html:38]: I'll leave this, but let's try a few different ones? Doubt it'll make a strong difference.

**Pull request merged**